packages/common/test/ArcGISContextManager.test.ts

```diff
@@ -1188,12 +1190,18 @@ describe("ArcGISContext:", () => {
         portalUrl: "https://qaext.arcgis.com",
       });
       expect(mgr.context.hubUrl).toBe("https://hubqa.arcgis.com");
+      expect(mgr.context.survey123Url).toEqual(
```

Comment:
Minor, the `it` text for this test reas `handles qaext huburl` but we're now testing more than the `hubUrl`, maybe rename the test to `handles qaext urls`. Similar point below for `handles devext hubUrl => handles devext urls`. We're additionally checking the values of certain URLs in the following tests:
* `verify props when passed just portalUrl`
* `verify props when passed session`
* `verify props after clearing session`
* `sign out on qa, resets portalUrl correctly`
* `sign out on dev, resets portalUrl correctly`

Maybe copy/paste a quick assertion into each those tests asserting the value of `survey123Url` too for parity, should be a quick hit.